AMW_business/src/main/java/ch/puzzle/itc/mobiliar/business/releasing/boundary/ResourceEntityDto.java

```diff
@@ -0,0 +1,37 @@
+package ch.puzzle.itc.mobiliar.business.releasing.boundary;
+
+import lombok.Getter;
+import lombok.Setter;
+
+public class ResourceEntityDto implements Comparable<ResourceEntityDto> {
+    @Getter
```

Comment:
You could probably annotate the class with 
`@Data ` and `@AllArgsConstructor` and save some lines